src/main/cpp/v8/JNIV8Object.cpp

```diff
@@ -79,7 +79,9 @@ void JNIV8Object::weakPersistentCallback(const WeakCallbackInfo<void>& data) {
     // IF we do, we have to make a strong reference to the java object again and also register this callback for
     // the provided JS object reference!
     jniV8Object->_jsObject.ClearWeak();
-    jniV8Object->_jsObject.MarkActive();
+
+    //TODO: replace deleted method.
+    //jniV8Object->_jsObject.MarkActive();
 
```

Comment:
Unclear what is going on here.

Reply:
MarkActive informed the GC that the object pointed to by the respective Handle should not be collected during the next sweep. It's basically like a "temporary retain". You would have to call this every sweep to keep objects alive that way - which is required for some of the more complex mechanisms that V8 supports (EmbedderHeapTracer). We are not using those; but I read that calling this can give a tiny gc speed boost because this reference does not need to be checked in the next sweep. Unclear how much effect if any this ever had.
NOT obligatory. So just removing this is fine.